### DISCUSSION TITLE
Sample MQTT payload/msg

### DISCUSSION THREAD

**Discussion by veerur4u** [jomjol/AI-on-the-edge-device#2598] [General]
Hi, we are planning to use AI on edge for one of our pilot project initiatives to monitor water meters and utility meters ,  and are in process of evaluating hardware needs and procurement.  In mean time we would like understand what will be the MQTT payload like, if some one could provide the sample MQTT payload for an water meter, that  the AI on edge is posting to the MQTT topic. Thanks for the help

**Comment by caco3** [comment]
MQTT is documented here: https://jomjol.github.io/AI-on-the-edge-device-docs/MQTT-API/

Also I would like to suggest to organize a device and let it run in demo mode, see https://jomjol.github.io/AI-on-the-edge-device-docs/Demo-Mode/#demo-mode. This way you could play with the config and see what you can get.

**Reply by veerur4u** [reply]
Thanks @caco3 . Will check this references , it helps.. At this point we don't have a device yet, we are planning build a simulator which can act like device and just post data to MQTT topic. This allows us to build the backend services while we are waiting for our hardware to be setup.
